doc/install.rst

```diff
@@ -0,0 +1,83 @@
+Build Instructions
```

Comment:
Personally, I would like to see the main build instructions in the main readme.md. This way, a user figures out how to build the project by looking at the front page.

What do you think?

Reply:
I think it's nicer to have these things on separate pages instead of having one monolithic README.  But I understand your point that it's nice if the user sees the relevant information directly on the repo main page.  Would adding a section to the README that simply links to the corresponding page be a viable alternative?

Reply:
yes, having a link to the build page from the main readme is good with me. Can you add this for this PR?

Reply:
done